openfisca_uk/parameters/consumption/fuel/prices/petrol.yaml

```diff
@@ -0,0 +1,16 @@
+description: Average price of unleaded petrol per litre.
```

Comment:
Could this cite the CPI instead? ONS mentions average fuel prices [here](https://www.ons.gov.uk/economy/inflationandpriceindices/bulletins/consumerpriceinflation/january2022) but I didn't see a dataset off hand.

If not, I think average prices could deserve its own folder within `parameters`, e.g. `parameters.average_prices.fuel.petrol`. When I first saw `prices` in the tree I thought it was some sort of set price.